src/app/modules/pages/tech-en_US/tech.component.html

```diff
@@ -0,0 +1,80 @@
+<ng-template #loading>
+  <app-container theme="light" fxFill>
+    <mat-spinner *ngIf="loading"></mat-spinner>
+  </app-container>
+</ng-template>
+<app-container *ngIf="navigation$ | async as navigation; else loading">
+  <div class="container-inner">
+    <app-column [gap]="36">
+      <app-column [gap]="16">
+        <em>
+          <a href="/tecnologia">Leia em Português</a>
```

Comment:
Que tal a gente especificar o que será traduzido?

```suggestion
          <a href="/tecnologia">Leia esta página em Português</a>
```

Reply:
Boa! Modifiquei aqui